test/Confluent.Kafka.UnitTests/Error.cs

```diff
@@ -27,6 +27,11 @@ public void Constuctor()
             var e = new Error(ErrorCode._BAD_COMPRESSION);
             Assert.Equal(e.Code, ErrorCode._BAD_COMPRESSION);
             Assert.NotNull(e.Message);
+            Assert.Equal(e.Message, "Local: Invalid compressed data");
```

Comment:
I realized I've been typically getting the order of arguments wrong in Assert.Equal. First argument is expected, last argument is actual. This is noted in messages when running the tests so we should change (because it's confusing otherwise). However, no need to fix here I think, I'll do a PR which changes it everywhere.